package.json

```diff
@@ -19,7 +19,7 @@
     "url": "https://github.com/estools/esrecurse.git"
   },
   "dependencies": {
-    "estraverse": "~4.1.0",
+    "estraverse": "~4.2.0",
```

Comment:
Sure, feel free to change. I believe those were just added by `npm install --save` way before `^` existed.